### PR TITLE
Better arrows in huts

### PR DESCRIPTION
**Pull request by Nightenom**
Don't merge till textures will be solved
### Changes proposed in this pull request:
- changed arrows and add page number system to all huts (is automatic for any number of pages)
- add onHover (display another element of given id) and on/off/isOn functions to Pane
- add dropfixx="{int}" to the dropdown element to fix X value in views with absolute X different than 0

Review please

### CODE REVIEW

**Comment by Raycoms** [comment]
It's a lot of GUI changes so I'm not sure if it works fine, did you test it with all changed GUIs?

**Comment by Nightenom** [comment]
I stated many times that I was not able to do it, but now I'm home so I will, I will probably do the textures myself since visuals seem to don't have time

**Comment by Nightenom** [comment]
The changes were introduced mostly for align attribute not working properly, at least I think it's not

**Comment by Nightenom** [comment]
Should be ready, the page handler for huts is done, the one for townhall will probably suffer one more rebuild to be completely generic (like the for huts)
Now you just use the `windowhutworkerplaceholder.xml` for any worker hut without having to write another page handler

File: src/main/java/com/minecolonies/coremod/client/gui/AbstractWindowBuilding.java [hunk 1]
```diff
@@ -24,12 +24,14 @@
      * Type B is a class that extends {@link AbstractBuildingWorker.View}.
      */
     protected final B          building;
```
Comment (LDTTeam-Buildsystem) [inline src/main/java/com/minecolonies/coremod/client/gui/AbstractWindowBuilding.java:25]: ### Errors:
- Protected field building 
- Can be package-private

### Warnings:
- Instance field name building doesn't match regex 'm_[a-z][A-Za-z\d]*'

File: src/main/java/com/minecolonies/coremod/client/gui/AbstractWindowBuilding.java [hunk 2]
```diff
@@ -24,12 +24,14 @@
...
-    private final   SwitchView switchView;
     private final   Label      title;
```
Comment (LDTTeam-Buildsystem) [inline src/main/java/com/minecolonies/coremod/client/gui/AbstractWindowBuilding.java:26]: ### Errors:
- Type of field 'title' is concrete class Label 

### Warnings:
- Instance field name title doesn't match regex 'm_[a-z][A-Za-z\d]*'

File: src/main/java/com/minecolonies/coremod/client/gui/AbstractWindowBuilding.java [hunk 3]
```diff
@@ -24,12 +24,14 @@
...
-    private final   Button     buttonPrevPage;
-    private final   Button     buttonNextPage;
     private final   Button     buttonBuild;
```
Comment (LDTTeam-Buildsystem) [inline src/main/java/com/minecolonies/coremod/client/gui/AbstractWindowBuilding.java:27]: ### Errors:
- Type of field 'buttonBuild' is concrete class Button 

### Warnings:
- Instance field name buttonBuild doesn't match regex 'm_[a-z][A-Za-z\d]*'

File: src/main/java/com/minecolonies/coremod/client/gui/AbstractWindowBuilding.java [hunk 4]
```diff
@@ -24,12 +24,14 @@
...
     private final   Button     buttonRepair;
```
Comment (LDTTeam-Buildsystem) [inline src/main/java/com/minecolonies/coremod/client/gui/AbstractWindowBuilding.java:28]: ### Errors:
- Type of field 'buttonRepair' is concrete class Button 

### Warnings:
- Instance field name buttonRepair doesn't match regex 'm_[a-z][A-Za-z\d]*'

File: src/main/java/com/minecolonies/coremod/client/gui/AbstractWindowBuilding.java [hunk 5]
```diff
@@ -24,12 +24,14 @@
...
+    protected       SwitchView switchView;
```
Comment (LDTTeam-Buildsystem) [inline src/main/java/com/minecolonies/coremod/client/gui/AbstractWindowBuilding.java]: ### Errors:
- Protected field switchView 
- Type of field 'switchView' is concrete class SwitchView 
- Can be package-private

### Warnings:
- Instance field name switchView doesn't match regex 'm_[a-z][A-Za-z\d]*'

**Review by LDTTeam-Buildsystem — DISMISSED** [review]
#### Analysis Complete
##### Statistics:
|        | Count | New | Old | Diff |
| ------ |:-----:|:---:|:---:|:----:|
| Total  | 39805 | 26  | 29  |  -3  |
| Errors |  601  |  0  |  0  |  0   |
##### More Information:
- [Analysis - Full](https://teamcity.minecolonies.com/viewLog.html?buildId=15764&tab=Inspection&buildTypeId=bt88)

File: src/main/java/com/minecolonies/coremod/client/gui/AbstractWindowBuilding.java [hunk 6]
```diff
@@ -24,12 +24,14 @@
...
+    protected       int        switchPagesSize;
```
Comment (LDTTeam-Buildsystem) [inline src/main/java/com/minecolonies/coremod/client/gui/AbstractWindowBuilding.java]: ### Errors:
- Protected field switchPagesSize 
- Can be package-private

### Warnings:
- Instance field name switchPagesSize doesn't match regex 'm_[a-z][A-Za-z\d]*'

File: src/main/java/com/minecolonies/coremod/client/gui/AbstractWindowBuilding.java [hunk 7]
```diff
@@ -24,12 +24,14 @@
...
+    protected final Label      pageNum;
```
Comment (LDTTeam-Buildsystem) [inline src/main/java/com/minecolonies/coremod/client/gui/AbstractWindowBuilding.java]: ### Errors:
- Protected field pageNum 
- Type of field 'pageNum' is concrete class Label 
- Can be package-private

### Warnings:
- Instance field name pageNum doesn't match regex 'm_[a-z][A-Za-z\d]*'

File: src/main/java/com/minecolonies/coremod/client/gui/AbstractWindowBuilding.java [hunk 8]
```diff
@@ -24,12 +24,14 @@
...
+    protected final Button     buttonPrevPage;
```
Comment (LDTTeam-Buildsystem) [inline src/main/java/com/minecolonies/coremod/client/gui/AbstractWindowBuilding.java]: ### Errors:
- Protected field buttonPrevPage 
- Type of field 'buttonPrevPage' is concrete class Button 
- Can be private

### Warnings:
- Instance field name buttonPrevPage doesn't match regex 'm_[a-z][A-Za-z\d]*'

File: src/main/java/com/minecolonies/coremod/client/gui/AbstractWindowBuilding.java [hunk 9]
```diff
@@ -24,12 +24,14 @@
...
+    protected final Button     buttonNextPage;
```
Comment (LDTTeam-Buildsystem) [inline src/main/java/com/minecolonies/coremod/client/gui/AbstractWindowBuilding.java]: ### Errors:
- Protected field buttonNextPage 
- Type of field 'buttonNextPage' is concrete class Button 
- Can be private

### Warnings:
- Instance field name buttonNextPage doesn't match regex 'm_[a-z][A-Za-z\d]*'

**Review by LDTTeam-Buildsystem — DISMISSED** [review]
#### Analysis Complete
##### Statistics:
|        | Count | New  | Old  | Diff |
| ------ |:-----:|:----:|:----:|:----:|
| Total  | 40273 | 2665 | 2197 | +468 |
| Errors |  605  |  5   |  1   |  +4  |
##### More Information:
- [Analysis - Full](https://teamcity.minecolonies.com/viewLog.html?buildId=16497&tab=Inspection&buildTypeId=bt88)

**Review by LDTTeam-Buildsystem — DISMISSED** [review]
#### Analysis Complete
##### Statistics:
|        | Count | New | Old | Diff |
| ------ |:-----:|:---:|:---:|:----:|
| Total  | 40273 |  0  |  0  |  0   |
| Errors |  605  |  0  |  0  |  0   |
##### More Information:

**Review by LDTTeam-Buildsystem — DISMISSED** [review]
#### Analysis Complete
##### Statistics:
|        | Count | New | Old | Diff |
| ------ |:-----:|:---:|:---:|:----:|
| Total  | 40273 |  0  |  0  |  0   |
| Errors |  605  |  0  |  0  |  0   |
##### More Information:

**Review by LDTTeam-Buildsystem — DISMISSED** [review]
#### Analysis Complete
##### Statistics:
|        | Count | New | Old | Diff |
| ------ |:-----:|:---:|:---:|:----:|
| Total  |   0   |  0  |  0  |  0   |
| Errors |   0   |  0  |  0  |  0   |
##### More Information:

**Review by LDTTeam-Buildsystem — DISMISSED** [review]
#### Analysis Complete
##### Statistics:
|        | Count | New | Old | Diff |
| ------ |:-----:|:---:|:---:|:----:|
| Total  |   0   |  0  |  0  |  0   |
| Errors |   0   |  0  |  0  |  0   |
##### More Information:

**Review by LDTTeam-Buildsystem — APPROVED** [review]
#### Analysis Complete
##### Statistics:
|        | Count | New | Old | Diff |
| ------ |:-----:|:---:|:---:|:----:|
| Total  |   0   |  0  |  0  |  0   |
| Errors |   0   |  0  |  0  |  0   |
##### More Information:

**Review by marchermans — APPROVED** [review]

**Pull request merged**